js/activity.js

```diff
@@ -3325,86 +3325,85 @@ class Activity {
                 );
                 return;
             }
-            activity._restoreTrash();
+        
+            if (docById("helpfulWheelDiv").style.display !== "none") {
+                docById("helpfulWheelDiv").style.display = "none";
+                activity.__tick();
+            }
+        };
+
+        const restoreTrashShortcut = (activity) => {
```

Comment:
Why "Shortcut"? Isn't this basically a "pop" from the FILO trash heap?

Reply:
Sir I changed it